getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -32,13 +37,17 @@ the JDK can be used for this purpose:
 
     keytool -keyalg RSA -genkeypair -alias androiddebugkey -keypass android -keystore debug.keystore -storepass android -dname "CN=Android Debug,O=Android,C=US" -validity 9999
 
+- This will create a ``debug.keystore`` file in your current directory. You should move it to a memorable location such as ``%USERPROFILE%\.android\``, because you will need it's location in a later step.
+  - see this Q&A for more info on the KeyTool process: https://godotengine.org/qa/21349/jdk-android-file-missing?show=21399#a21399
```

Comment:
I think it would be better to integrate the relevant points from this QA answer directly in the docs. It mostly duplicates the existing content but makes it more Windows-friendly, so this should be changed in the main doc to make it friendly to all platforms.

Reply:
the Q&A has more detail than minimally required, so updating to say that.